proofs/signatures/th_lira.plf

```diff
@@ -66,3 +70,119 @@
          ((>=_IntReal x y) (>=_Real (reify_int_term x) (reify_real_term y)))
          (default (fail formula))
          ))
+
+; From an arith predicate, prove the equivalent real predicate
+(declare pf_reified_arith_pred
+  (! p formula
+  (! p' formula
+    (! (th_holds p)
```

Comment:
I thought there was an identifier missing between the `!` and the `(th_holds p)` but this file type checks. How come?

Reply:
Interesting! Perhaps LFSC regards those identifiers as optional? I'm actually not sure.

Reply:
I'll put the identifier in there!

Reply:
Sounds good. Thanks.
Maybe @ajreynol can tell us what's going on with missing identifiers type-checking in lfsc? :)